File: kikiboba.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    string word;
    int b{0}, k{0};
    
    cin >> word;
    
    for (int i=0; i < word.length(); ++i)
    {
        if (word[i] == 'b')
        {
            b++;
        }
        
        if (word[i] == 'k')
        {
            k++;
        }
    }
    
    if (b == 0 && k == 0)
    {
        cout << "none";
    }
    else if (b > k)
    {
        cout << "boba";
    }
    else if (b < k)
    {
        cout << "kiki";
    }
    else
    {
        cout << "boki";
    }
    
    return 0;
}

File: kikiboba.py
word = input()

k = 0
b = 0

for x in word:
    if x == 'k':
        k = k + 1
        
    if x == 'b':
        b = b + 1
        
if k == 0 and b == 0:
    print("none")
elif k > b:
    print("kiki")
elif b > k:
    print("boba")
elif k is b:
    print("boki")
